tox.ini

```diff
@@ -165,14 +165,12 @@ deps = gitpython >= 2.1.10
        oauth2client >= 4.1.2
 commands = python {toxinidir}/tasks/notify.py
 
-[testenv:dev]
-description = generate a DEV environment
+[testenv:devenv]
+description = create '{toxworkdir}/devenv' with all dependencies for development
 extras = testing, docs
 deps = {[testenv]deps}
        {[testenv:release]deps}
        {[testenv:notify]deps}
-# required to make looponfail reload on every source code change
 usedevelop = True
-basepython = python3.7
 commands = python -m pip list --format=columns
-           python -c 'import sys; print(sys.executable)'
+           python -c 'import sys; print("devenv at: %s" % sys.exec_prefix)'
```

Comment:
We don't use the ``%`` operator but prefer format instead. That being said paired with the fact that I also find env redundant I'm going to close this. Also having the python executable directly I think it's better because that's what you need to provide to the IDE as the interpreter to use.